developer/app-router/installation.mdx

```diff
@@ -135,11 +135,14 @@ This catch-all route will fetch page data from `Makeswift` and pass it to be ren
 
 <CatchAllExample />
 
-<Note>
-  Make sure to remove the root page component at `src/app/page.tsx` file. This
-  will ensure all pages are managed by Makeswift. Otherwise, you'll get a
-  specificity error on your routes.
-</Note>
+Delete the root page component `src/app/page.tsx` file to ensure that all pages (including the home page) are managed by Makeswift.
```

Comment:
What do you think if we put this before the example, together with the other sentence? Also, maybe we should specify that this isn't just about managing all pages in Makeswift but also that Next.js will blow up if you have both an optional catch-all route and a page at the level.

Maybe above we could say "Delete the root page (i.e., `app/page.tsx`) and Create an optional catch-all route named `[[...path]]`. "